File: C++/3512.min_op_to_mk_arr_divisible_by_k.cpp
class Solution {
public:
    int minOperations(vector<int>& nums, int k) {
        int sum=0,count=0;
       for(int i=0;i<nums.size();i++){
        sum+=nums[i];
       } 
       while(sum%k!=0){
        if(sum%k!=0){
            sum=sum-1;
            count++;
        }
       }
       return count;
    }
};

File: Python3/3512.min_op_to_mk_arr_divisible_by_k.py
from typing import List

class Solution:
    def minOperations(self, nums: List[int], k: int) -> int:
        sum = 0
        count = 0
        for i in range(len(nums)):
            sum += nums[i]
        while sum % k != 0:
            sum -= 1
            count += 1
        return count